### PR TITLE
HSC-240: Modified drug list

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket :https://mekomsolutions.atlassian.net/browse/HSC-240

### CODE REVIEW

File: configuration/drugs/drugs.csv [hunk 1]
```diff
@@ -68,8 +69,11 @@ c9d9540c-80e0-49ee-8819-44811ecbb23e,,Charbon activé 260mg,ebe04ba6-768f-42e5-9
...
 4b6a0eea-3793-4d51-85f3-37c3e3fa3264,,Ciprofloxacin 500mg,a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,fdbe907b-6abc-41dc-9599-2d84105e4063,500mg,
 de257e4d-6efd-4fd7-8626-1e9da1d92d1a,,Ciprofloxacin 3mg/ml,a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,7c46524c-a14f-43b6-b4b7-792cee31d8d5,3mg/ml,
-043fd8b5-bdc1-4709-8152-44b99e1d433d,,Cisatracurium 10mg/ml,a67db321-80d4-41f0-be6a-24f50d11c804,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,10mg/ml,
+2a986874-fc59-486d-ba7e-06c84af38a31,,Ciprofloxacin 500mg/100ml,a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,500mg/100ml,
+043fd8b5-bdc1-4709-8152-44b99e1d433d,TRUE,Cisatracurium 10mg/ml,a67db321-80d4-41f0-be6a-24f50d11c804,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,10mg/ml,
 d23311cd-8ba0-440e-8aeb-5edf028a654f,,Clarithromycin 500mg,a4391c57-549b-4047-8971-c3fa72e140b5,fdbe907b-6abc-41dc-9599-2d84105e4063,500mg,
+49f6b8b6-7a5a-440f-8e0f-3680fa25813a,,Clindamycine 150mg Capsule,bdfbb1ec-59e1-4c5b-8c0c-15c743cf5d9e,164cd72a-b50d-41f5-b534-7baff3e434b9,150mg,
+9b323866-2da1-4be7-9ba4-d10fcd79f95e,,Clindamycine  150mg Injectable,bdfbb1ec-59e1-4c5b-8c0c-15c743cf5d9e,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,150mg,
```
Comment (rbuisson) [inline configuration/drugs/drugs.csv]:
```suggestion
9b323866-2da1-4be7-9ba4-d10fcd79f95e,,Clindamycine 150mg Injectable,bdfbb1ec-59e1-4c5b-8c0c-15c743cf5d9e,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,150mg,
```

**Comment by kazlaw** [comment]
All the newly added drugs and those with modifications load successfully against prod data. No conflicts or failing concepts have been observed in iniz logs
image

image

File: configuration/drugs/drugs.csv [hunk 2]
```diff
@@ -287,35 +296,37 @@ c109ae6a-9002-43ce-87c8-b1d29d5dd661,,Eucalyptus & Diphenhydramine & Phenylephri
...
 17f3e82f-279b-4657-bcb9-6361ad59e0e7,,Silver Nitrate Pencil 75%,984e4058-d87f-489f-a28d-138c90565666,0689bb6e-9371-4a0f-ad1c-63c135b1714d,75%,
 10a9b381-7926-459c-94ff-c81440430fbf,,Acide Ascorbique (Vit C) 1000mg,b7bab21f-5d84-4e48-b056-bdc25e696365,fdbe907b-6abc-41dc-9599-2d84105e4063,1000mg,
-9241d8ee-2fa4-4965-a6ca-8426beffe8a2,,Acide borique & Antipyrine & Ephedrine 33.65mg/ml,a546446e-e789-4d02-b02f-c938c6bd373f,7c46524c-a14f-43b6-b4b7-792cee31d8d5,33.65mg/ml,
-de4c409b-7d37-48a7-aecc-302189d4ab03,,Acide borique & Antipyrine & Benzocaine 80mg/ml,07470e7f-f959-4344-89f6-8e6a0f442bef,7c46524c-a14f-43b6-b4b7-792cee31d8d5,80mg/ml,
-54e2bf22-e7dc-48d5-9ee1-8132e1a354c7,,Acide tranexamique 1g/4ml,c6557538-cc7e-4ca8-bfeb-80ff5c4f4976,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,1g/4ml,
+9241d8ee-2fa4-4965-a6ca-8426beffe8a2,,"Acide borique, Antipyrine, Ephedrine (ophtalnol) 33.65mg/ml",a546446e-e789-4d02-b02f-c938c6bd373f,7c46524c-a14f-43b6-b4b7-792cee31d8d5,33.65mg/ml,
+de4c409b-7d37-48a7-aecc-302189d4ab03,,"Acide borique, Antipyrine, Benzocaine (othorinol) 80mg/ml",07470e7f-f959-4344-89f6-8e6a0f442bef,7c46524c-a14f-43b6-b4b7-792cee31d8d5,80mg/ml,
```
Comment (rbuisson) [inline configuration/drugs/drugs.csv]: We should keep consistency. Don't we use `&` to separate active ingredients?

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/drugs/drugs.csv (hunk 2 above)
Comment (kazlaw) [reply configuration/drugs/drugs.csv]: > We should keep consistency. Don't we use `&` to separate active ingredients?

This has been fixed.

**Pull request merged**